public/js/test/utils/agent.js

```diff
@@ -0,0 +1,94 @@
+module.exports = function() {
```

Comment:
I'll probably rename Agent to something more descriptive.

I'll also add comments.